core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -328,5 +378,18 @@ object Props extends Logger {
         Map()
     }
   }
+
+  private var prepended:List[java.util.Map[Object, Object]] = Nil
+  private var appended:List[java.util.Map[Object, Object]] = Nil
+  private var interpolators:List[java.util.Map[Object, Object]] = Nil
```

Comment:
Fwiw I'm not sure he was suggesting we use an immutable map to begin with, just that we use a Scala one. I'm down for that (`mutable.Map` rather than `java.util.Map`).